VideoLocker/res/values/styles.xml

```diff
@@ -113,6 +113,15 @@
         <item name="android:background">@color/edx_brand_secondary_base</item>
     </style>
 
+    <style name="hairline_separator_style">
```

Comment:
btw there are alot of styles in the file with `style` suffix. So, should I do this change or not ?